### PR TITLE
adding list

### PR DESCRIPTION
**Pull request by admiralAwkbar**
- Adding the `-r` to pass a list or repos

### CODE REVIEW

File: gh-repo-stats [hunk 1]
```diff
@@ -1492,9 +1525,14 @@
 #### Function ValidateInput ####################################################
 ################################################################################
 ValidateInput() {
-  if [[ $OUTPUT != "Table" ]] && [[ $OUTPUT != "CSV" ]]; then
-    echo "Output parameter (-O, --output) '${OUTPUT}' is invalid. Must be 'CSV' or 'Table'"
-    exit 1;
+  # Check if REPO_LIST is empty
+  if [[ -n "${REPO_LIST}" ]]; then
```
[the remark this answers is not in the document]
Comment (ssulei7) [reply gh-repo-stats]: We can ignore this.

File: gh-repo-stats [hunk 2]
```diff
@@ -1492,9 +1525,14 @@
...
+    # Put the contents of the file into an array
+    IFS=',' read -r -a REPO_LIST_ARRAY <<< "${REPO_LIST}"
```
[the remark this answers is not in the document]
Comment (ssulei7) [reply gh-repo-stats]: Not sure if we should clean this up, at least when we reference the list.

**Review by ssulei7 — APPROVED** [review]
Otherwise, looks good to me unless others have objections.

File: gh-repo-stats [hunk 3]
```diff
@@ -133,6 +134,10 @@
       ORG_NAME=$2
       shift 2
       ;;
+    -r|--repo-list)
```
[the remark this answers is not in the document]
Comment (admiralAwkbar) [reply gh-repo-stats]: fixed to `rl`

File: gh-repo-stats [hunk 4]
```diff
@@ -789,22 +795,37 @@
...
     REPO_NAME=$(_jq '.name' | tr '[:upper:]' '[:lower:]')
 
+    #########################################################
+    # Need to see if the user gave a repo list to filter on #
+    #########################################################
+    if [[ -n "${REPO_LIST}" ]]; then
+      # convert REPO_NAME to lowercase
+      CHECK_REPO_NAME=$(echo "${REPO_NAME}" | tr '[:upper:]' '[:lower:]')
+      # Check if the repo name is in the list
+      for REPO in '${REPO_LIST_ARRAY[@]}'; do
```
[the remark this answers is not in the document]
Comment (admiralAwkbar) [reply gh-repo-stats]: Fixed the quotes

**Pull request merged**